onboarding-app/NavigationOptionView.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import UIKit
+
+final class NavigationOptionView: UIStackView {
+    
+    required init(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    required init(
+        firstLineText: String,
+        secondLineText: String,
+        imageSystemName: String,
+        viewController: Any? = nil,
```

Comment:
better to have a closure like `onTap` instead of vc. but before changing anything let's run the memory profile and check if there is memory leaks